src/main/java/org/weasis/dicom/google/api/ui/SearchPanel.java

```diff
@@ -43,10 +44,12 @@
 
 import static javax.swing.BoxLayout.LINE_AXIS;
 import static javax.swing.BoxLayout.PAGE_AXIS;
+import static javax.swing.BoxLayout.X_AXIS;
 
 public class SearchPanel extends JPanel {
 
     private final JTextField patientName = textField();
+    private final JCheckBox fuzzyMatching = textBox("fuzzy match", true);
```

Comment:
The fuzzy match checkbox should be false by default.

Reply:
done